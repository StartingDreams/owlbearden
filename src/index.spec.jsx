import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<div />, div);
});

it('div renders correctly', () => {
  const tree = renderer.create(<div>test</div>).toJSON();
  expect(tree).toMatchSnapshot();
});
