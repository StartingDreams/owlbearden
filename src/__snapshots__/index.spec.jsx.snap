// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`div renders correctly 1`] = `
<div>
  test
</div>
`;
